translation.rst

```diff
@@ -1073,6 +1073,27 @@ of:
         }
     }
 
+.. tip::
+
+    If you need to get the locale used in the callback of the
+    ``runWithLocale()`` method, you can declare the ``$locale`` special
+    argument like this::
```

Comment:
I think it's not a "special" argument. Now callback receive the locale as first argument.

We can maybe just update example below with `string $locale` ?

Reply:
I reworded, I think it's better now :+1: